app/src/main/java/cm/aptoide/pt/app/view/AppViewFragment.java

```diff
@@ -380,7 +380,7 @@
     listDonateButton = view.findViewById(R.id.donate_button);
     LinearLayoutManager linearLayoutManager = new LinearLayoutManager(getContext()) {
       @Override public boolean canScrollVertically() {
-        return false;
+        return true;
```

Comment:
This will be the same as the default, can we remove this implementation?
inside we have:
` @Override
    public boolean canScrollVertically() {
        return mOrientation == VERTICAL;
    }`

Reply:
👍